File: Second Year (2013-2014)/OMGL7/TP2 - Website in PHP/musiconline/musiconline/db/Titre.php
<?php
/* Authors: ANDRIAMILANTO Tompoariniaina && CANNO Camille
* Groupe: B1
* File: Titre.php
*/

class Titre implements DataObject{
   private $IdTitre;
   private $Libelle;
   private $Prix;
   private $Artiste;
   private $TitreAlbum;
   private $TitreCategorie;

   public function  __construct() { }

   public function init($IdTitre,$Libelle,$Prix,$Artiste,$TitreAlbum,$TitreCategorie){
      $this->IdTitre=$IdTitre;
      $this->Libelle=$Libelle;
      $this->Prix=$Prix;
      $this->Artiste=$Artiste;
      $this->TitreAlbum=$TitreAlbum;
      $this->TitreCategorie=$TitreCategorie;
   }


   public function getIDTitre(){
     return $this->IdTitre;
   }

   public function setIDTitre($IdTitre){
     $this->IdTitre = $IdTitre;
   }


   public function getLIbelle(){
     return $this->Libelle;
   }

   public function setLIbelle($Libelle){
     $this->Libelle = $Libelle;
   }


   public function getPRix(){
     return $this->Prix;
   }

   public function setPRix($Prix){
     $this->Prix = $Prix;
   }


   public function getARtiste(){
     return $this->Artiste;
   }

   public function setARtiste($Artiste){
     $this->Artiste = $Artiste;
   }


   public function getTItreAlbum(){
     return $this->TitreAlbum;
   }

   public function setTItreAlbum($TitreAlbum){
     $this->TitreAlbum = $TitreAlbum;
   }


   public function getTItreCategorie(){
     return $this->TitreCategorie;
   }

   public function setTItreCategorie($TitreCategorie){
     $this->TitreCategorie = $TitreCategorie;
   }


}
?>

File: Second Year (2013-2014)/OMGL7/TP3 - Website in JSP/src/db/Titre.java
package db;
/**
*@author Andriamilanto Tompoariniaina - Canno Camille - Info 2 B
*/

/**
*Classe pour les objets Titre
*/
public class Titre {
//Attributs
	private int idTitre;
	private String libelle;
	private float prix;
	private String artiste;
	private int titreAlbum;
	private int titreCategorie;

//Methodes
	/**
	*Constructeur
	*/
	public Titre() {}

	/**
	*Methode pour initialiser les attributs
	*/
	public void init(int idti, String lib, float price, String art, int titreAl, int titreCat) {
		this.idTitre = idti;
		this.libelle = lib;
		this.prix = price;
		this.artiste = art;
		this.titreCategorie = titreCat;
		this.titreAlbum = titreAl;
	}

//Accesseurs
	public void setIdTitre(int idti) {
		this.idTitre = idti;
	}

	public int getIdTitre() {
		return this.idTitre;
	}

	public void setLibelle(String lib) {
		this.libelle = lib;
	}

	public String getLibelle() {
		return this.libelle;
	}

	public void setPrix(float price) {
		this.prix = price;
	}

	public float getPrix() {
		return this.prix;
	}

	public void setArtiste(String art) {
		this.artiste = art;
	}

	public String getArtiste() {
		return this.artiste;
	}

	public void setTitreAlbum(int title) {
		this.titreAlbum = title;
	}

	public int getTitreAlbum() {
		return this.titreAlbum;
	}

	public void setTitreCategorie(int tit) {
		this.titreCategorie = tit;
	}

	public int getTitreCategorie() {
		return this.titreCategorie;
	}
}
